Move SDKGracefulTermination To Stable (#3231)
googleforgames/agones: site/content/en/docs/Guides/Client SDKs/_index.md

Be aware that if you use a variation of `System.exit(0)` after calling SDK.Shutdown(), your game server container may
restart for a brief period, inline with our [Health Checking]({{% ref "/docs/Guides/health-checking.md#health-failure-strategy" %}}) policies. 

> If the `SDKGracefulTermination` feature is enabled, when the SDK server receives the TERM signal before calling SDK.Shutdown(),
> the SDK server would stay alive for the period of the terminationGracePeriodSeconds until SDK.Shutdown() has been called

Reviewer: Just some cleanup since the gate doesn't exist, and also since we're here.

```suggestion
If the SDK server receives a TERM signal before calling SDK.Shutdown(),
the SDK server will stay alive for the period of the `terminationGracePeriodSeconds` until `SDK.Shutdown()` has been called. 
```

Author: made the changes as per your suggestion
